code/datums/spells/cluwne.dm

```diff
@@ -13,7 +13,11 @@
 
 /mob/living/carbon/human/proc/makeCluwne()
 	to_chat(src, "<span class='danger'>You feel funny.</span>")
-	adjustBrainLoss(80)
+	if(!get_int_organ(/obj/item/organ/internal/brain/cluwne))
+		var/obj/item/organ/internal/brain/cluwne/idiot_brain = new
+		idiot_brain.insert(src, make_cluwne = 0)
+		idiot_brain.dna = dna.Clone()
```

Comment:
Are you sure this won't cause weirdness with having more than one brain, or making anticluwning no longer work?